modules/ROOT/pages/_partials/data-sync/client-using-a-data-sync-client-to-query-a-mutation-on-a-data-sync-server.adoc

```diff
@@ -0,0 +1,101 @@
+[id="client-using-a-data-sync-client-to-query-a-mutation-on-a-data-sync-server-{context}"]
+= Using a Data Sync Client to query a mutation on a Data Sync Server
+
+Build a Data Sync Server and query the mutation.
+The Data Sync Server will be running locally in this procedure.
+
+.Prerequisites
+
+* The example in link:https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/_partials/data-sync/server-create-a-graphql-schema-for-an-existing-data-source.adoc[Getting started with an example data source schema] runs locally.
+* Basic knowledge of `npm`.
+
+.Procedure
+
+. Create a basic link:https://webpack.js.org/guides/getting-started/[_webpack project_] repo.
++
+. Open a terminal and navigate into the cloned project.
++
+. To add the necessary Data Sync Client javascript libraries to your basic _webpack project_, execute the following commands:
++
+[source,bash]
+----
+npm install @aerogear/voyager-client
+npm install graphql-tag
+----
++
+. Before continuing, build the project and ensure that it successfully builds by executing `npm run build` - the output should be similar to:
++
+[source,bash]
```

Comment:
unnecessary

Reply:
"npm run build" threw an error for me.

`0 info it worked if it ends with ok
1 verbose cli [ '/Users/bdooley/.nvm/versions/node/v10.15.3/bin/node',
1 verbose cli   '/Users/bdooley/.nvm/versions/node/v10.15.3/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build' ]
2 info using npm@6.9.0
3 info using node@v10.15.3
4 verbose stack Error: missing script: build
4 verbose stack     at run (/Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/lib/run-script.js:155:19)
4 verbose stack     at /Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/lib/run-script.js:63:5
4 verbose stack     at /Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/read-package-json/read-json.js:115:5
4 verbose stack     at /Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/read-package-json/read-json.js:418:5
4 verbose stack     at checkBinReferences_ (/Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/read-package-json/read-json.js:373:45)
4 verbose stack     at final (/Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/read-package-json/read-json.js:416:3)
4 verbose stack     at then (/Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/read-package-json/read-json.js:160:5)
4 verbose stack     at ReadFileContext.<anonymous> (/Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/read-package-json/read-json.js:332:20)
4 verbose stack     at ReadFileContext.callback (/Users/bdooley/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:90:16)
4 verbose stack     at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:237:13)
5 verbose cwd /Users/bdooley/Documents/Code/webpack-demo
6 verbose Darwin 18.6.0
7 verbose argv "/Users/bdooley/.nvm/versions/node/v10.15.3/bin/node" "/Users/bdooley/.nvm/versions/node/v10.15.3/bin/npm" "run" "build"
8 verbose node v10.15.3
9 verbose npm  v6.9.0
10 error missing script: build
11 verbose exit [ 1, true ]
`

Reply:
After talking with @briandooley - it was suggested that the issue here could relate to the version of node/npm. The following versions allowed this to build on my local machine:

[jstaffor@jstaffor mobile-docs]$ npm -version
6.9.0

[jstaffor@jstaffor mobile-docs]$ node -v
v8.12.0